FantasmoSDK/app/src/test/java/com/fantasmo/sdk/config/RemoteConfigTest.kt

```diff
@@ -43,4 +50,16 @@ class RemoteConfigTest {
             imageQualityFilterModelVersion = "0.1.0"
         )
     }
+    @Test
+    fun testRemoteConfigNullId() {
+        RemoteConfig.updateConfig(InstrumentationRegistry.getInstrumentation().context, "null")
+        assertEquals(RemoteConfig.remoteConfig.remoteConfigId, remoteConfig.remoteConfigId)
+    }
+
+    @Test
+    fun testRemoteConfigEmptyArray() {
```

Comment:
it's a PITA but could we name the test functions after what we actually want them to test, i.e. in this case `fun testUsesOldConfigWhenRemoteConfigIdIsArray`

Reply:
Technically this test checks that the old remoteConfigId is used (it could be that other values were updated and this test would miss that). Although having a single assert per test is normally a good rule of thumb, there are some situations where it's just impractical to do so - I think this one qualifies. Let's also assert that another one of the old values is the same, or if you want to go god-mode, a test helper that populates an array/set and asserts checks the sets are equal 😈